frontend/app/(editing)/draft/d/[slug]/page.tsx

```
@@ -7,7 +7,12 @@ import { getTags } from "@/lib/requests/tag";
 import { NextSteps } from "@/components/draft/metadata/next-steps/next-steps";
 import { Overview } from "@/components/draft/metadata/overview";
 import { Description } from "@/components/draft/metadata/description";
-import { isContentEditor } from "@/lib/utils";
+import {
+  isAuthorizedUserAdmin,
```

ESLint

[error] 'isAuthorizedUserAdmin' is defined but never used.

(@typescript-eslint/no-unused-vars)